packages/client/hmi-client/src/services/workflow.ts

```diff
@@ -32,7 +32,13 @@ export const create = () => {
 	return workflow;
 };
 
-export const addNode = (wf: Workflow, op: Operation, pos: Position) => {
+type NodeSize = { width: number; height: number };
+export const addNode = (
+	wf: Workflow,
+	op: Operation,
+	pos: Position,
+	size: NodeSize = { width: 180, height: 220 }
+) => {
```

Comment:
Either add a `Size` at the end of `src/types/workflow.ts`
```ts
export interface Size {
	width: number;
  height: number;
}
```

or update your function to 

```ts
export const addNode = (
	wf: Workflow,
	op: Operation,
	pos: Position,
	size: { width: WorkflowNode["width"], height: WorkflowNode["height"] } = { width: 180, height: 220 }
) => {
```

Reply:
i dont understand what needs to be done here, lets keep track of this and address it in another pr

Reply:
I commited the changes